### PR TITLE
CMT4K_wiki: Update.

### PR DESCRIPTION
**Pull request by lzc731524299**
- Match update.
- Tournament event date error fix.

### CODE REVIEW

File: wiki/Tournaments/CMT_4K/2017/en.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ The **Chinese osu!mania 4K Tournament 2017** (***CMT 4K 2017***) is the second o
 | Drawings | 25 June 2017 (20:00 UTC+8) |
 | Group Stage | 30 June - 2 July 2017 |
 | Round of 16 | 8-9 July 2017 |
-| Round of 8 | 14-15 July 2017 |
+| Round of 8 | 15-16 July 2017 |
 | Semifinals | 22-23 July 2017 |
 | Finals - Week 1 | 29-30 July 2017 |
 | Finals - Week 2 | 5-6 Aug 2017 |
```
Comment (TPGPL) [inline wiki/Tournaments/CMT_4K/2017/en.md]: Add dots after Day in dates

File: wiki/Tournaments/CMT_4K/2017/en.md [hunk 2]
```diff
@@ -9,15 +9,15 @@ The **Chinese osu!mania 4K Tournament 2017** (***CMT 4K 2017***) is the second o
...
-| Round of 8 | 14-15 July 2017 |
-| Semifinals | 22-23 July 2017 |
-| Finals - Week 1 | 29-30 July 2017 |
-| Finals - Week 2 | 5-6 Aug 2017 |
+| Registration | 4. - 18. June 2017 |
+| Qualification Stage | 24. 25. June 2017 |
+| Drawings | 25. June 2017 (20:00 UTC+8) |
+| Group Stage | 30. June - 2. July 2017 |
+| Round of 16 | 8. 9. July 2017 |
+| Round of 8 | 15. 16. July 2017 |
```
Comment (TPGPL) [inline wiki/Tournaments/CMT_4K/2017/en.md]: 15-16. ?

File: wiki/Tournaments/CMT_4K/2017/en.md [hunk 3]
```diff
@@ -9,15 +9,15 @@ The **Chinese osu!mania 4K Tournament 2017** (***CMT 4K 2017***) is the second o
...
-| Qualification Stage | 24-25 June 2017 |
-| Drawings | 25 June 2017 (20:00 UTC+8) |
-| Group Stage | 30 June - 2 July 2017 |
-| Round of 16 | 8-9 July 2017 |
-| Round of 8 | 14-15 July 2017 |
-| Semifinals | 22-23 July 2017 |
-| Finals - Week 1 | 29-30 July 2017 |
-| Finals - Week 2 | 5-6 Aug 2017 |
+| Registration | 4. - 18. June 2017 |
+| Qualification Stage | 24. 25. June 2017 |
```
Comment (TPGPL) [inline wiki/Tournaments/CMT_4K/2017/en.md]: 24-25.

File: wiki/Tournaments/CMT_4K/2017/en.md [hunk 4]
```diff
@@ -9,15 +9,15 @@ The **Chinese osu!mania 4K Tournament 2017** (***CMT 4K 2017***) is the second o
...
-| Registration | 4-18 June 2017 |
-| Qualification Stage | 24-25 June 2017 |
-| Drawings | 25 June 2017 (20:00 UTC+8) |
-| Group Stage | 30 June - 2 July 2017 |
-| Round of 16 | 8-9 July 2017 |
-| Round of 8 | 14-15 July 2017 |
-| Semifinals | 22-23 July 2017 |
-| Finals - Week 1 | 29-30 July 2017 |
-| Finals - Week 2 | 5-6 Aug 2017 |
+| Registration | 4. - 18. June 2017 |
```
Comment (TPGPL) [inline wiki/Tournaments/CMT_4K/2017/en.md]: 4-18. ?

File: wiki/Tournaments/CMT_4K/2017/en.md [hunk 5]
```diff
@@ -9,15 +9,15 @@ The **Chinese osu!mania 4K Tournament 2017** (***CMT 4K 2017***) is the second o
...
+| Qualification Stage | 24. 25. June 2017 |
+| Drawings | 25. June 2017 (20:00 UTC+8) |
+| Group Stage | 30. June - 2. July 2017 |
+| Round of 16 | 8. 9. July 2017 |
+| Round of 8 | 15. 16. July 2017 |
+| Semifinals | 22. 23. July 2017 |
+| Finals - Week 1 | 29. 30. July 2017 |
```
Comment (TPGPL) [inline wiki/Tournaments/CMT_4K/2017/en.md]: 29-30. ?

File: wiki/Tournaments/CMT_4K/2017/en.md [hunk 6]
```diff
@@ -9,15 +9,15 @@ The **Chinese osu!mania 4K Tournament 2017** (***CMT 4K 2017***) is the second o
...
+| Finals - Week 2 | 5. 6. August 2017 |
```
Comment (TPGPL) [inline wiki/Tournaments/CMT_4K/2017/en.md]: 5-6. ?

**Comment by lzc731524299** [comment]
I'm not familiar with the English date styling, thanks for the advice.

**Pull request merged**